addons/google_drive/i18n/google_drive.pot

```diff
@@ -15,6 +15,12 @@ msgstr ""
 "Content-Transfer-Encoding: \n"
 "Plural-Forms: \n"
 
+#. module: google_drive
+#: code:addons/google_drive/models/google_drive.py:0
+#, python-format
+msgid " You can disable the feature in the General Settings"
```

Comment:
It seems *pretty unlikely* that people on v13 will be updating the translations for gdrive, if we even translate those.

Reply:
It's to make @mart-e happy.